TileFormats/BatchTable/README.md

```diff
@@ -167,342 +158,6 @@ var geographicArray = new Float64Array(batchTableBinary.buffer, byteOffset, geog
 var geographicOfFeature = positionArray.subarray(batchId * numberOfComponents, batchId * numberOfComponents + numberOfComponents); // Using subarray creates a view into the array, and not a new array.
 ```
 
-## Batch Table Hierarchy
-
-The standard batch table is suitable for datasets composed of features with the same sets of properties. However, some datasets have more complex metadata structures such as feature types or feature hierarchies that are not easy to represent as parallel arrays of properties. The Batch Table Hierarchy provides more flexibility for these cases.
```

Comment:
It might be worthwhile to keep just this section but point to the extension for more details.